src/util/ConfigManager/Validator.h

```diff
@@ -98,6 +98,10 @@ class ConfigOptionValidatorManager {
   // A descripton of the invariant, this validator imposes.
   std::string descriptor_;
 
+  // Describes the order of initialization.
+  static inline size_t numberOfInstances_{0};
+  size_t initializationId_{numberOfInstances_++};
+
```

Comment:
Same here, use `atomic<size_t>`

Reply:
Done, see [9d86c3d](https://github.com/ad-freiburg/qlever/pull/1153/commits/9d86c3ddbae62e2bea452201b9009fecf8b4317c).

How would the non hacky way look like? I thought of saving the date and time, but I'm not sure if those values would be precise enough.